### ISSUE TITLE
Multiple input files

### ISSUE THREAD

**Issue by elimpnick** [ben-strasser/fast-cpp-csv-parser#74]
Hi,

I was wondering if there's a built-in way to open multiple (same layout) csv files into one CSVReader object?  I have 2 (sometimes 3) csv files that have the exact same column structure that I would like to open into one concatenated CSVReader object.  

I am trying to avoid concatenating the csv files on disk prior to reading them with CSVReader and I'd like to avoid having 2-3 individual CSVReader objects to handle.

Basically, I'm wondering if there's another input option (say a vector of strings) instead of a direct filename such as below:

`io::CSVReader<8, io::trim_chars<' '>, io::double_quote_escape<',', '\"'> > in(Filename);`

Thanks!

**Comment by ben-strasser** [comment]
Hi,

there is no built in way.

What I have personally done in the past to achieve this was to implement 
was to implement my own ByteSource. The derived class essentially 
contained an ifstream, a counter, and a basename. The class first tried 
to open basename + to_string(counter). If it succeeded, it read the 
content, incremented the counter, and then tried to open basename + 
to_string(counter). If opening failed, it returned 0 to indicate an EOF.

(The class also additionally implemented on-the-fly zip decompression 
but that is beside the point of the question.)

Best Regards
Dr. Ben Strasser

**Comment by elimpnick** [comment]
I see, thanks for the answer and the great library!
